### PR TITLE
[ART-1856] Adding request live ids to advisories job

### PR DESCRIPTION
**Pull request by shiywang**
For more details discussion, see https://issues.redhat.com/browse/ART-1856

https://localhost:8888/job/hack/job/shiywang-aos-cd-jobs/job/build%252Fadvisories/14/console

image

### CODE REVIEW

**Comment by sosiouxme** [comment]
I'd like to suggest two tweaks to make this simpler:
1. Do this as part of the advisories job - AFAIK there's no reason we need to wait in order to request live ids (I thought it had to be in QE state but it seems not).
2. Only request for the image (payload) advisory. We need that one for creating the release, the others can be created by docs whenever they sign off.

File: jobs/build/advisories/Jenkinsfile [hunk 1]
```diff
@@ -184,6 +197,32 @@ node {
...
+                        Hello docs team,I'd like to request Live IDs for our ${params.VERSION}.z advisories:\n
+                        RPM: https://errata.devel.redhat.com/advisory/${rpm_advisory_id}\n
+                        Image: https://errata.devel.redhat.com/advisory/${image_advisory_id}\n
+                    """
+                    if (params.VERSION.startsWith("4")) {
+                        live_id_content += """
+                        Extras: https://errata.devel.redhat.com/advisory/${extras_advisory_id}\n
+                        OLM Operators metadata: https://errata.devel.redhat.com/advisory/${metadata_advisory_id}\n
+                        """
+                    }
```
Comment (thiagoalessio) [inline jobs/build/advisories/Jenkinsfile:212]:
```suggestion
                    def live_id_content = """
                        Hello docs team,I'd like to request Live IDs for our ${params.VERSION}.z advisories:\n
                        Image: https://errata.devel.redhat.com/advisory/${image_advisory_id}\n
                    """

Comment (sosiouxme) [reply jobs/build/advisories/Jenkinsfile:212]: Well, uh....
`Hello docs team, ART would like to request a Live ID for our ${params.VERSION}.z advisory:\n`

Comment (shiywang) [reply jobs/build/advisories/Jenkinsfile:212]: Sure, ART is better, I just bumped from jira template

**Comment by shiywang** [comment]
@thiagoalessio @sosiouxme PTAL

**Pull request merged**